contracts/wrapper/NameWrapper.sol

```diff
@@ -514,9 +542,10 @@ contract NameWrapper is
      * @param node The namehash of the name to check fuses on.
      * @param fuseMask A bitmask of fuses that must not be burned.
      */
-    modifier operationAllowed(bytes32 node, uint96 fuseMask) {
-        (, uint96 fuses) = getData(uint256(node));
-        if (fuses & fuseMask != 0) {
+
+    modifier operationAllowed(bytes32 node, uint32 fuseMask) {
+        (, uint32 fuses, uint64 expiry) = getData(uint256(node));
+        if (fuses & fuseMask != 0 && expiry > block.timestamp) {
```

Comment:
Doesn't `getData` already set fuses to 0 if we're after the expiry?

Reply:
Yes this is true. I'll remove `expiry > block.timestamp`